src/posit/connect/metrics/visits.py

```diff
@@ -153,6 +153,7 @@ def find(self, *args, **kwargs) -> List[VisitEvent]:
     @overload
     def find_one(
         self,
+        *,
```

Comment:
same question about the places we're adding `*`

Reply:
This is to say, all arguments after `*` must be provided as keyword arguments. So, the user must call `find_one(content_guid="asdf")`. `find_one("asdf")` will throw an error.

Reply:
This is what we want since positional argument can change over time.